components/mentions/Mentions.razor.cs

```diff
@@ -92,14 +101,34 @@ private void SetClassMap()
                 .Add(prefixCls)
                 .If($"{prefixCls}-disable", () => Disable)
                 .If($"{prefixCls}-focused", () => Focused)
-                .If($"{prefixCls}-rtl", () => RTL)
-                ;
+                .If($"{prefixCls}-rtl", () => RTL);
         }
 
         protected override void OnInitialized()
         {
             base.OnInitialized();
             SetClassMap();
+            InitializePrefixes();
+        }
+
+        protected override void OnParametersSet()
+        {
+            base.OnParametersSet();
+            InitializePrefixes();
+        }
+
+        private void InitializePrefixes()
+        {
+            _prefixes = (Prefix ?? "@")
+                .Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
```

Comment:
Use `Split(',' , StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries)` so that we do not need to handle the trimming manually next step.